crates/libcontainer/src/container/container_kill.rs

```diff
@@ -95,7 +103,14 @@ impl Container {
         let pids = cmanger.get_all_pids()?;
         pids.iter().try_for_each(|&pid| {
             log::debug!("kill signal {} to {}", signal, pid);
-            signal::kill(pid, signal)
+            let res = signal::kill(pid, signal);
+            match res {
+                Err(nix::errno::Errno::ESRCH) => {
+                    /* the process does not exist, which is what we want */
+                    Ok(())
+                }
+                _ => res,
+            }
```

Comment:
Now thinking, I'm not sure if we need this, as we are freezing the cgroup before getting all pids, so whatever pids we get, the process must exist, as freezing cgroup will suspend all processes in the cgroup